recipe/meta.yaml

```diff
@@ -1,74 +1,74 @@
+{% set version = "2020.1" %}
+{% set build_number = 0 %}
+{% set sha256 = "7513f7054f4eeb5361de1f5113883145fc67dbabde73a2148f221ae05af4d22c" %}
 
 package:
-    name: {{ name|lower }}
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    fn: {{ name }}-{{ version }}.tar.gz
-    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: {{ sha256 }}
-    patches:
-        - osx_flags.diff
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - osx_flags.diff
 
 build:
-    number: {{ build_number }}
+  number: {{ build_number }}
 
 requirements:
-    build:
-        - {{ compiler('cxx') }}
-        - {{ compiler('m2w64_cxx') }}  # [win and vc<14]
-    host:
-        - python
-        - libpython  # [win and vc<14]
-        - setuptools
-        - pybind11
-        - numpy
-        - ocl-icd  # [linux]
-        - khronos-opencl-icd-loader  # [win or osx]
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('m2w64_cxx') }}      # [win and vc<14]
+  host:
+    - python
+    - libpython      # [win and vc<14]
+    - setuptools
+    - pybind11
+    - numpy
+    - ocl-icd      # [linux]
+    - khronos-opencl-icd-loader      # [win or osx]
 
-    run:
-        - python
-        - {{ pin_compatible("numpy") }}
-        - pytools
+  run:
+    - python
+    - {{ pin_compatible("numpy") }}
+    - pytools
         # for pkg_resources
-        - setuptools
-        - six
-        - mako
-        - appdirs
-        - khronos-opencl-icd-loader  # [win or osx]
-        - ocl_icd_wrapper_apple  # [osx]
-        - ocl-icd  # [linux]
+    - setuptools
+    - six
+    - mako
+    - appdirs
+    - khronos-opencl-icd-loader      # [win or osx]
+    - ocl_icd_wrapper_apple      # [osx]
+    - ocl-icd      # [linux]
 
 test:
-    requires:
-        - pytest
-        - pocl  >=0.14  # [linux]
-    imports:
-        - {{ name }}
-    source_files:
-        - test
-    commands:
+  requires:
+    - pytest
+    - pocl  >=0.14      # [linux]
+  imports:
+    - {{ name }}
+  source_files:
+    - test
+  commands:
         # apple implementation on 10.14 is buggy. So skip test_can_build_and_run_binary
-        - cd test && python -m pytest -k "not test_can_build_and_run_binary"   # [osx]
+    - cd test && python -m pytest -k "not test_can_build_and_run_binary"       # [osx]
 
 about:
-    home: http://pypi.python.org/pypi/pyopencl
-    doc_url: https://documen.tician.de/pyopencl
-    dev_url: http://github.com/inducer/pyopencl
-    license: MIT
-    license_file: LICENSE
-    summary: Python wrapper for OpenCL
-    description: |
+  home: http://pypi.python.org/pypi/pyopencl
+  doc_url: https://documen.tician.de/pyopencl
+  dev_url: http://github.com/inducer/pyopencl
+  license: MIT
+  license_file: LICENSE
+  summary: Python wrapper for OpenCL
+  description: |4
```

Comment:
@beckermr, any idea where this 4 came from?

Reply:
Yikes. Nope.

Reply:
We may want to stop the bot if this persists. Cc @CJ-Wright

Reply:
ok